App.js

```diff
@@ -100,40 +98,35 @@ const CommonHome = () => {
 
 const App = ({userStore}) => {
   const [onboarded, setOnboarded] = useState();
-  useEffect(() => {
-    const loadUser = async () => {
-      try {
-        if (!userStore.userInfo) {
-          console.log('LOAD USER 1');
-          const googleSignedInUser = await authService.getGoogleSignedInUser();
-          // Signed In Mode
-          if (googleSignedInUser) {
-            userStore.setIsLoading(true);
-            const appUser = await firebaseService.getUserById(
-              googleSignedInUser.user.id,
-            );
 
-            const allUserInfo = {
-              ...googleSignedInUser.user,
-              ...appUser,
-            };
+  const onAuthStateChanged = async user => {
+    try {
+      userStore.setIsLoading(true);
+      if (user) {
+        const appUser = await FirebaseService.getInstance().getUserById(
+          user.uid,
+        );
 
-            const filteredUser = filterObjectByKeys(
-              allUserInfo,
-              userInfoFields,
-            );
-            userStore.setSignedInUser(filteredUser);
-            userStore.setIsLoading(false);
-          }
-          // Anonymous mode
-          else {
-            console.log('Anonymous user');
-          }
-        }
-      } catch (error) {
-        console.log('ERRROR', error);
+        const allUserInfo = {
+          ...user._user,
+          ...appUser,
+        };
+
+        const filteredUser = filterObjectByKeys(allUserInfo, userInfoFields);
+
+        userStore.setSignedInUser(filteredUser);
+        userStore.setIsLoading(false);
```

Comment:
this line is not needed, we have this in l. 122

Reply:
This line is actually removed in this PR. It's in red.

Reply:
Hm, strange on the code view, this labels are appearing in red code 

![image](https://user-images.githubusercontent.com/26001304/78995989-4eeb5d00-7b4c-11ea-835c-4622a633a333.png)